addons/l10n_tr_nilvera_einvoice/models/account_edi_xml_ubl_tr.py

```diff
@@ -151,6 +211,26 @@ def _get_additional_document_reference_list(self, invoice):
             })
         return additional_document_reference_list
 
+    def _get_invoice_line_price_vals(self, line):
+        # EXTEND 'account.edi.common'
+        invoice_line_price_vals = super()._get_invoice_line_price_vals(line)
+        invoice_line_price_vals['base_quantity_attrs'] = {'unitCode': self._get_uom_unspsc_code(line)}
+
+        return invoice_line_price_vals
+
+    def _get_invoice_line_vals(self, line, line_id, taxes_vals):
+        invoice_line_vals = super()._get_invoice_line_vals(line, line_id, taxes_vals)
+        invoice_line_vals['line_quantity_attrs'] = {'unitCode': self._get_uom_unspsc_code(line)}
+        return invoice_line_vals
+
+    def _get_uom_unspsc_code(self, line):
```

Comment:
Can't you just override the `_get_uom_unece_code` method in this module?
I'm not sure it works but I would expect it to.